tutorials/access-mac-mini-with-reemo/index.mdx

```diff
@@ -17,7 +17,7 @@ dates:
 ## Reeemo - Overview
 
 [Reemo](https://reemo.io/) is an ultra-low latency, high performance desktop platform designed to allow anyone to connect to any computer and virtual environment from a Chrome browser to work and play freely from anywhere, on any device. 
-Reemo works perfectly on MacOS,Linux, and Windows and supports multiple peripherals. _Reemo is a French startup that was part of the Scaleway Startup Program._
+Reemo works perfectly on MacOS, Linux, and Windows and supports multiple peripherals. _Reemo is a French startup that was part of the Scaleway Startup Program._
```

Comment:
```suggestion
Reemo works perfectly on MacOS, Linux, and Windows and supports multiple peripherals. _Reemo is a French startup that participates in the [Scaleway Startup Program](https://www.scaleway.com/en/startup-program/)._
```

Reply:
According to the Startup Program confluence page, Reemo no longer participates in the Startup Program.